src/drivers/kinetis/tone_alarm/tone_alarm.cpp

```diff
@@ -481,6 +425,7 @@ void ToneAlarm::next_note()
 	}
 
 	// and arrange a callback when the note should stop
+	assert(duration != 0);
```

Comment:
Can you please avoid using `assert`?

Reply:
They are removed. I was going to replace it with a log and exit and I found out 0 is a possible return. :(